src/parcsr_ls/par_mgr_setup.c

```diff
@@ -617,7 +612,7 @@ hypre_MGRSetup( void               *mgr_vdata,
    /* Allocate memory for level structure */
    if (A_array == NULL)
    {
-      A_array = hypre_CTAlloc(hypre_ParCSRMatrix*, max_num_coarse_levels, HYPRE_MEMORY_HOST);
+      A_array = hypre_CTAlloc(hypre_ParCSRMatrix*, max_num_coarse_levels + 1, HYPRE_MEMORY_HOST);
```

Comment:
Still wrapping my head around this one vs using the RAP_ptr, but it is not a big deal.

Reply:
This saves me some lines of code for computing MGR statistics in `par_mgr_stats.c`